tests/publiccloud/run_ltp.pm

```diff
@@ -58,7 +58,7 @@ sub run {
 
     my $provider = $self->provider_factory();
     my $instance = $self->{my_instance} = $provider->create_instance();
-    $instance->wait_for_guestregister();
+    $instance->wait_for_guestregister() if get_var('FLAVOR') !~ m/BYOS/;
```

Comment:
you can do it. But it isn't nessesary as guestregister is available on BYOS images and is inactive as well, see: https://openqa.suse.de/tests/4081611#step/run_ltp/79

Reply:
well idea from my side was that this `if` will save us from extra ssh call when we know in advance the result :)

Reply:
I know. "inactive" is also a return condition, but why not avoiding this step if not necessary?

Reply:
But if we call it, then we actually check that the guest register successfully get inactive. And is not missing or failed.

Reply:
ok , let's keep calling it